list/__test__/list.test.js

```diff
@@ -12,12 +12,24 @@ describe('List Data Structure', () => {
     expect(stuff.data).toEqual({});
   });
 
-  it('pushes items to the end of the data set', () => {
+  it('pops item off end of array', () => {
     let stuff = new List();
     stuff.push('a');
     stuff.push('b');
     expect(stuff.length).toEqual(2);
-    expect(stuff.data[1]).toEqual('b');
+    stuff.pop();
```

Comment:
On line 20 we call `stuff.pop()` and since `'b'` is the last thing our list it should remove it. So why down below on line 22 is `stuff.pop` returning `'b'` when we supposedly removed it on line 20? (This test is passing for the wrong reason)

I'll let you try to figure it out first and then I can explain it if needed.